### ISSUE TITLE
batch size and lr for multiple gpu

### ISSUE THREAD

**Issue by liyunsheng13** [RangiLyu/nanodet#156]
Hi
The config file e.g. nanode-m.yml is for model trained with one gpu. I wonder if the learning rate and batch size need to be changed when I try to train the model with eight gpus. Btw, how many gpu do you use to achieve the reported results?

Thanks

**Comment by RangiLyu** [comment]
Lr and batch size are setting for a single distributed training process on each gpu. You just need to change the gpu_ids when you switch to multiple gpus. Also remember to adjust the workers_per_gpu because the total number of dataloader workers will increase with the number of gpus.

I trained these models with one gpu.
